docs/concepts/controllers.md

```diff
@@ -0,0 +1,110 @@
+# Controllers
+
+etcd-druid is an operator to manage etcd clusters, and follows the [`Operator`](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) pattern for Kubernetes.
+It makes use of the [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) framework which makes it quite easy to define Custom Resources (CRs) such as `Etcd`s and `EtcdCopyBackupTask`s through [*Custom Resource Definitions*](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) (CRDs), and define controllers for these CRDs.
+etcd-druid uses Kubebuilder to define the `Etcd` CR and its correspdonding controllers.
+<!--- the autogenerated API documents are to be linked for the etcd CR  -->
```

Comment:
are you going to remove this comment ?

Reply:
This was for generation of API docs for `etcd-druid` through the `gen-crd-api-reference-docs`tool. Will remove this comment for now, and will add the api document generator in a future PR.